app/test_main.py

```diff
@@ -1,3 +1,69 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (16, 16, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (30, 20, [3, 1])
+    ],
```

Comment:
It seems like you are missing some edge cases for the test. For example, you could test what happens when the age of the cat or dog is negative. This scenario is not mentioned in the task description, but it would be a good practice to cover such cases in the tests.